config/src/cli.rs

```diff
@@ -430,6 +440,47 @@ impl Clone for RelayerConfig {
     }
 }
 
+/// Wraps an encryption key (public or private) to allow for the relayer to be
+/// configured with either of the encryption key or the decryption key
+///
+/// Configuring the relayer with the decryption key allows the relayer to
+/// automate tasks like fee redemption
+#[derive(Copy, Clone, Debug, Serialize, Deserialize)]
+pub enum RelayerFeeKey {
```

Comment:
nit: we use this type throughout the codebase, would be more discoverable to put it in `common::types` than here in `config::cli`

Reply:
i thought about this, but don't want `types` to become a dumping ground for anything used in multiple places. so we can move it if it needs a more general location.